source/include/cellular_api.h

```diff
@@ -639,6 +639,78 @@ CellularError_t Cellular_SocketRegisterClosedCallback( CellularHandle_t cellular
                                                        CellularSocketClosedCallback_t closedCallback,
                                                        void * pCallbackContext );
 
+/**
+ * @brief Configure parameters of an SSL Context.
+ *
+ * @param[in] cellularHandle The opaque cellular context pointer created by Cellular_Init.
+ * @param[in] sslContextId The SSL context ID.
+ * @param[in] sslConfigurationParameter The SSL parameter to be configured.
+ * @param[in] inputArg The value to be passed to the SSL parameter.
+ *
+ * @return CELLULAR_SUCCESS if the operation is successful, otherwise an error
+ * code indicating the cause of the error.
+ */
+CellularError_t Cellular_ConfigureSSLContext(CellularHandle_t cellularHandle,
```

Comment:
watch out for the formatting, in the previous declarations there's a blank space after the opening curved bracket. i.e. `( `

Reply:
Also, the arguments are column aligned